src/index.html

```diff
@@ -1,13 +1,375 @@
+    <link rel="icon" type="image/x-icon" href="./images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header" id="home">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-links">
+            <a href="#menu" class="icon icon--menu">
+            </a>
+            
+            <a href="#home" class="logo">
+              <img 
+              src="./images/logo.png"
+              alt="miami logo" 
+              class="logo__img">
+            </a>
+            
+            <a href="tel:+1 234 555-55-55" class="icon icon--phone header__call">
+            </a>
+          </div>
+          <h1 class="header__title">Miami<br>Condo Kings</h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-links">
+            <a href="#home" class="icon icon--cross">
+            </a>
+            
+            <a href="#home" class="logo">
+              <img 
+              src="./images/logo.png"
+              alt="miami logo" 
+              class="logo__img">
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+          <a href="tel:+1 234 555-55-55" 
+             class="menu__call menu__call--phone">+1 234 555-55-55
+          </a>
+
+          <a href="tel:+1 234 555-55-55" 
+             class="menu__call menu__call--call">call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+    <main class="main">
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">Miami best<br>Real estate
+                <a href="#home" 
+                  class="about__link link" 
+                  target="_blank">here
+                </a>
+              </h2>
+            </div>
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__slogan link"
+              target="_blank"
+              >Over 50 years experience in love.</a>
```

Comment:
```suggestion
              <a 
                href="#" 
                class="about__slogan link"
                target="_blank"
              >
                Over 50 years experience in love.
              </a>
```

Check your code style in whole project